UoaWDCC/younite-web
# 109 Projects Image Component Styling

## Context
Fixing up the project image component and adding styling to fit the Figma design. 

Closes #109 

## What Changed?
- Added styling for both active and past projects
- Fixed timeline project type usage

## How To Review
- Check project images match Figma design
- View next\components\projects\ProjectsImage.tsx and projects\Timeline.tsx

## Notes
Further increasing the height of the images for the active projects breaks the timeline, so the height doesn't match Figma exactly
